AudioKit/Common/Internals/AKSettings.swift

```diff
@@ -222,7 +226,9 @@ extension AKSettings {
 
         // Activate session
         do {
-            try session.setActive(true)
+            try AKTry({
+                try session.setActive(true)
+            })
```

Comment:
Since this a trailing closure, can't you just do 

```
AKTry {
    try session.setActive(true)
}
```

Reply:
I don't think so because I added a `finally` optional closure. Swift was balking at that. However I never end up using that anyway, so I could probably simplify it.

Reply:
I never use finally.